App/src/App.tsx

```diff
@@ -51,6 +51,7 @@ export function App() {
                         path={BaseRoutes.Home}
                         element={isLoggedIn == true ? <Home /> : <Navigate to={BaseRoutes.Login} />}
                     />
+                    <Route path={BaseRoutes.CourseBrowser} element={<CourseBrowser />} />
```

Comment:
This should also be changed to the following to prevent non-logged-in users from visiting the course browser page:
  ```ts
  <Route
      path={BaseRoutes.CourseBrowser}
      element={isLoggedIn == true ? <CourseBrowser /> : <Navigate to={BaseRoutes.Login} />}
 />
```

Reply:
Good point, I added the conditional as you suggested.